### PR TITLE
Fix use GITHUB_OUTPUT from deprecated set-output

### LINKED ISSUE #2491: Fix set-output in Github Actions to be deprecated
Hi!

https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
As per this URL, set-output of Github Actions is deprecated.

I would like to create a PR for a fix to this issue.
Thanks.

### PR DESCRIPTION
**Pull request by k1rnt**
resolved #2491

### CODE REVIEW

File: .github/workflows/tests.yml [hunk 1]
```diff
@@ -48,8 +48,8 @@ jobs:
     # Get values for cache paths to be used in later steps 
     - id: cache-paths
       run: |
-        echo "::set-output name=go-cache::$(go env GOCACHE)"
-        echo "::set-output name=go-mod-cache::$(go env GOMODCACHE)"
+        echo "go-cache=$(go env GOCACHE) >> $GITHUB_OUTPUT"
```
Comment (gmlewis) [inline .github/workflows/tests.yml:51]:
```suggestion
        echo "go-cache=$(go env GOCACHE)" >> $GITHUB_OUTPUT
```

File: .github/workflows/tests.yml [hunk 2]
```diff
@@ -48,8 +48,8 @@ jobs:
...
+        echo "go-mod-cache=$(go env GOMODCACHE) >> $GITHUB_OUTPUT"
```
Comment (gmlewis) [inline .github/workflows/tests.yml:52]:
```suggestion
        echo "go-mod-cache=$(go env GOMODCACHE)" >> $GITHUB_OUTPUT
```

**Review by gmlewis — CHANGES_REQUESTED** [review]
Thank you, @k1rnt !
Just a couple minor tweaks, please.

**Comment by k1rnt** [comment]
Sorry.
Thanks for noticing all the details!

**Comment by gmlewis** [comment]
Hmmm... I'm not sure why the tests are failing... it looks like this change is not working properly.  The error message is:

```
Error: Path Validation Error: At least one directory or file path is required
```

ksnip_20221012-101850

Can you please investigate, @k1rnt and see if you can find out what is going on?

**Comment by k1rnt** [comment]
@gmlewis 
It appears that Shell needed to be specified correctly.
I've PASSED all the TESTs, so please re-review!

**Comment by gmlewis** [comment]
> @gmlewis It appears that Shell needed to be specified correctly. I've PASSED all the TESTs, so please re-review!

Wheh!  Thank you for your persistence, @k1rnt !  It is greatly appreciated!
:heart:

**Review by gmlewis — APPROVED** [review]

**Pull request merged**